### PR TITLE
Task-6 logout 

### PR DESCRIPTION
**Pull request by Jakubowski13**
It is created to check the functionality. not for merge now

### CODE REVIEW

File: src/containers/header/index.js [hunk 1]
```diff
@@ -52,21 +57,41 @@ const Header = () => {
...
+            ) : (
+              <>
+                <div className={styles.button} onClick={() => openModalLog()}>
+                  Log in
+                </div>
+                <div className={styles.button} onClick={() => openModalReg()}>
+                  Register
+                </div>
+              </>
+            )}
```
Comment (slavasgod) [inline src/containers/header/index.js:74]: Fix position changing of "Courses" and "Categories" links after login

File: src/containers/header/index.js [hunk 2]
```diff
@@ -52,21 +57,41 @@ const Header = () => {
...
           </div>
         </>
       ) : (
-        <Burger log={() => openModalLog()} reg={() => openModalReg()} />
+        <Burger
+          log={() => openModalLog()}
+          reg={() => openModalReg()}
+          isAuth={isAuthenticated}
+        />
```
Comment (slavasgod) [inline src/containers/header/index.js:82]: There are no logout button in burger menu after login.

File: src/containers/header/index.js [hunk 3]
```diff
@@ -52,21 +57,41 @@ const Header = () => {
...
       )}
       <Login modalStatus={isOpenLog} onModalClose={closeLoginModal} />
       <Register modalStatus={isOpenReg} onModalClose={closeRegModal} />
     </header>
   );
 };
 
-export default Header;
+const mapStateToProps = state => ({ props: state });
```
Comment (OlegPanasyuk) [inline src/containers/header/index.js:90]: Are you want to push all state to props???

**Comment by Jakubowski13** [comment]
created Task-6-v2-drop-down-Log-out-UI-Front-end

but auth don-t work correctly now to do logout

**Pull request closed without merging**